File: HW3-DynamicTetris/HW3-DynamicTetris/driver2.cpp
#include "tetris.hpp"
#include "tetromino.hpp"

using namespace std;

int main()
{
    int i = 0;
    char choice;
    int rowsize, columnsize;
    cout<<"Enter the row size";
    cin>>rowsize;
    cout<<"Enter the column size";
    cin>>columnsize;
    rowsize += 2;
    columnsize += 2;
    Tetris *tetris_ptr = new Tetris(rowsize, columnsize);
    
    do
    {
        tetris_ptr->move();
        cout<<"if you want to quit please enter q or if you want to continue enter any letter"<<endl;
        cin>>choice;
        if(choice == 'q' || choice == 'Q') return 0;

    } while (1);

    return 0;

}

File: HW7-TetrisJava/HW7-TetrisJava/src/Driver2.java
import java.util.Scanner;

/**
 *  Runs the tetris game with user inputs.
 * @author BeyzaAcar
 */

public class Driver2
{
    public static void main(String[] args)
    {
        int row, column;
        char block_Type;
        Scanner scan = new Scanner(System.in);
        System.out.println("Please enter the row ..:");
        row = scan.nextInt();
        System.out.println("Please enter the column ..:");
        column = scan.nextInt();
        Tetris tetrisObj = new Tetris(row+3, column+3);
        while(true)
        {
            System.out.println("enter the type of the tetromino you want to add ..:");
            block_Type = scan.next().charAt(0);
            tetrisObj.set_last_block(block_Type);
            tetrisObj.animate();
        }
    }
}



